src/Raven.Server/Web/System/AdminStorageHandler.cs

```diff
@@ -37,5 +38,37 @@ await using (var writer = new AsyncBlittableJsonTextWriterForDebug(context, Serv
                 writer.WriteEndObject();
             }
         }
+
+        [RavenAction("/admin/debug/storage/environment/scratch-buffer-info", "GET", AuthorizationStatus.Operator, IsDebugInformationEndpoint = false)]
+        public async Task SystemScratchBufferPoolInfoReport()
+        {
+            var env = ServerStore._env;
+
+            using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
+            await using (var writer = new AsyncBlittableJsonTextWriterForDebug(context, ServerStore, ResponseBodyStream()))
+            {
+                writer.WriteStartObject();
+                writer.WritePropertyName("Environment");
+                writer.WriteString("Server");
+                writer.WriteComma();
+
+                writer.WritePropertyName("Type");
+                writer.WriteString(nameof(StorageEnvironmentWithType.StorageEnvironmentType.System));
+                writer.WriteComma();
+
+                using (ServerStore.Engine.ContextPool.AllocateOperationContext(out ClusterOperationContext ctx))
+                using (var tx = env.ReadTransaction())
+                using (ctx.OpenWriteTransaction())
```

Comment:
I don't follow? Why read & write here?

Reply:
No need for both; that was fixed in a new PR